mobile/src/components/my-courses/my-courses.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from 'react';
+
+import { BackButton } from '~/components/common/common';
+import { useAppNavigate, useEffect } from '~/hooks/hooks';
+import { MyCourses as MyCoursesScreen } from '~/navigation/my-courses/my-courses.navigation';
+
+const MyCourses: FC = () => {
+  const navigation = useAppNavigate();
+
+  useEffect(() => {
+    navigation.setOptions({
+      headerLeft: () => (
+        <BackButton onPress={(): void => navigation.goBack()} />
```

Comment:
check that it will redirect user to the right screen, if not try to use pop()

Reply:
pop is usable only for Stack navigator, not for drawer(